dealroom_firestore_connector/__init__.py

```diff
@@ -194,6 +201,116 @@ def get_history_doc_refs(
     return tuple(doc.reference for doc in docs)
 
 
+def _validate_dealroomid(dealroom_id: Union[str, int]):
+    if not str(dealroom_id).isnumeric() and dealroom_id != -1:
+        raise ValueError("'dealroom_id' must be an integer")
```

Comment:
```suggestion
        raise ValueError("'dealroom_id' must be -1")
```
it only passes if it's -1, right?

Reply:
also, does this mean that function `set_history_doc_ref` should not be used with a payload that has `dealroom_id != -1`?

Reply:
This means that if the dealroom_Id is not numeric it will raise an error. Unfortunately, `.isnumeric` doesn't include negative numbers, so we do this extra check ourselves. So if you give anything that is not a number like or a -1 this will raise an error